gradle.properties

```diff
@@ -0,0 +1,2 @@
+checkstyleMainMaxWarnings=9122
+checkstyleTestMaxWarnings=416
```

Comment:
I added the thresholds to a separate file to make it easier to update them automatically in a forthcoming PR.